analytical_engine/java/README.md

```diff
@@ -37,9 +37,6 @@ enables Java programmers to write and run graph algorithms with these benefits.
 
 ### Building from source
 
-First you need **fastFFI** installed, see [alibaba/fastFFI](https://github.com/alibaba/fastFFI) for
-installation guide.
```

Comment:
Have we removed the need for manually install fastffi?

Reply:
Yes,  Grape-JDK now depends on FastFFI 0.1.2 from maven central repo, I have verified in this [PR](https://github.com/alibaba/GraphScope/pull/2028/files#diff-3e26a960618f1a1e05b2ceef90bb05079c02039c50af028f0f230fd3cc77039eL35)